1.6/Dockerfile

```diff
@@ -1,33 +1,50 @@
 FROM debian:jessie-backports
 
 RUN apt-get update \
-	&& apt-get install -y libssl1.0.0 libpcre3 lua5.3 --no-install-recommends \
+	&& apt-get install -y --no-install-recommends \
+		liblua5.3-0 \
+		libpcre3 \
+		libssl1.0.0 \
 	&& rm -rf /var/lib/apt/lists/*
 
 ENV HAPROXY_MAJOR 1.6
 ENV HAPROXY_VERSION 1.6.11
 ENV HAPROXY_MD5 844da4b553c887833550a008692e7a74
 
-# see http://sources.debian.net/src/haproxy/1.5.8-1/debian/rules/ for some helpful navigation of the possible "make" arguments
-RUN buildDeps='curl gcc libc6-dev liblua5.3-dev libpcre3-dev libssl-dev make' \
-	&& set -x \
+# see http://sources.debian.net/src/haproxy/jessie/debian/rules/ for some helpful navigation of the possible "make" arguments
+RUN set -x \
+	\
+	&& buildDeps=' \
+		gcc \
+		libc6-dev \
+		liblua5.3-dev \
+		libpcre3-dev \
+		libssl-dev \
+		make \
+		wget \
+	' \
 	&& apt-get update && apt-get install -y $buildDeps --no-install-recommends && rm -rf /var/lib/apt/lists/* \
-	&& curl -SL "http://www.haproxy.org/download/${HAPROXY_MAJOR}/src/haproxy-${HAPROXY_VERSION}.tar.gz" -o haproxy.tar.gz \
-	&& echo "${HAPROXY_MD5}  haproxy.tar.gz" | md5sum -c \
+	\
+	&& wget -O haproxy.tar.gz "http://www.haproxy.org/download/${HAPROXY_MAJOR}/src/haproxy-${HAPROXY_VERSION}.tar.gz" \
+	&& echo "$HAPROXY_MD5 *haproxy.tar.gz" | md5sum -c \
 	&& mkdir -p /usr/src/haproxy \
 	&& tar -xzf haproxy.tar.gz -C /usr/src/haproxy --strip-components=1 \
 	&& rm haproxy.tar.gz \
-	&& make -C /usr/src/haproxy \
+	\
```

Comment:
Yes, we like to have some whitespace to separate sections of the RUN line. The Dockerfile parser does currently ignore empty lines, but it was "fixed" to not do so in https://github.com/docker/docker/pull/24725 and reverted because of https://github.com/docker/docker/issues/29005 in https://github.com/docker/docker/pull/29064, but still planned to be deprecated with https://github.com/docker/docker/pull/29161.  As far as I know, comments in RUN lines will stay even if empty lines get removed.